code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -32,7 +32,7 @@
 	darksight = 3
 //	ambiguous_genders = TRUE
 	gluttonous = 1
-	slowdown = 0.3
+	slowdown = 0.15 // OCCULUS EDIT - Normalizing slowdown values
```

Comment:
Please put this back. Unathi have bonus health and are more difficult to buckle to things. The slowdown is to compensate for those strengths.